runtime/jcl/common/java_lang_invoke_MethodHandleNatives.cpp

```diff
@@ -811,7 +960,9 @@ Java_java_lang_invoke_MethodHandleNatives_resolve(
 	const J9InternalVMFunctions *vmFuncs = vm->internalVMFunctions;
 	jobject result = NULL;
 	J9UTF8 *name = NULL;
+	char nameBuffer[256];
```

Comment:
initializing the buffers will prevent unexpected issues
```suggestion
	char nameBuffer[256] = {0};
```